apis/r/R/utils-tiledb.R

```diff
@@ -18,3 +18,14 @@ map_query_layout <- function(layout) {
            tolower(layout)
     )
 }
+
+#' show_package_versions
+#'
+#' @description Prints package information suitable for assisting with bug reports.
+#'
+#' @export
+show_package_versions <- function() {
+    cat("tiledbsoma:   ", toString(utils::packageVersion("tiledbsoma")), "\n")
+    cat("tiledb:       ", toString(utils::packageVersion("tiledb")), "\n")
+    cat("R:            ", R.Version()$version.string, "\n")
+}
```

Comment:
Suggest to change to

```r
#' show_package_versions
#'
#' @description Prints package information suitable for assisting with bug reports.
#'
#' @export
#' @importFrom utils packageVersion
show_package_versions <- function() {
    cat("tiledbsoma:   ", toString(utils::packageVersion("tiledbsoma")), "\n")
    cat("tiledb:       ", toString(utils::packageVersion("tiledb")), "\n")
    cat("tiledb core:  ", tiledb::tiledb_version(TRUE), "\n")
    cat("R:            ", R.version.string, "\n")
}
```

(untested)

Reply:
@eddelbuettel added in latest commit

@thetorpedodog I added the core-version line to the Python side as well (great idea!)

Reply:
Cool, thanks, still haven't had a chance to run it so not sure we need `toString()`, part of me thinks that formatting should be automagic in the context.   

Oh, and I just checked, we already have `importFrom(utils,packageVersion)` in `NAMESPACE` so my add in what is now line 27 above wasn't needed (but won't hurt).

Reply:
All good @eddelbuettel , I ran it :)